plugins/node.d/amavis

```diff
@@ -18,11 +20,11 @@ The configuration environment variables are available
 
 =item amavislog
 
-Path to logfile (Default: "/var/log/mail/mail.info")
+Path to logfile (Default: "/var/log/maillog")
```

Comment:
I think, the old as well as the new value are both not universally applicable (at least here on Debian it would be `/var/log/mail.log`). Thus please leave this value unchanged if you do not have a good reason for a change.